src/uwtools/logging.py

```diff
@@ -36,3 +58,12 @@ def setup_logging(quiet: bool = False, verbose: bool = False) -> None:
         **({"filename": os.devnull} if quiet else {}),
     }
     logging.basicConfig(**kwargs)
+
+
+def use_logger(logger: logging.Logger) -> None:
+    """
+    Log hereafter via the given logger.
+
+    :param logger: The logger to log to.
+    """
+    log.logger = logger
```

Comment:
This is the public method to be used by API users to pass in a logger customized for their needs.